Library/Formula/offline-imap.rb

```diff
@@ -1,9 +1,8 @@
-require "formula"
-
 class OfflineImap < Formula
   homepage "http://offlineimap.org/"
-  url "https://github.com/OfflineIMAP/offlineimap/archive/v6.5.6.tar.gz"
-  sha1 "cced67060be2922ca3ce056839b2aa1fe6b776a1"
+  desc "OfflineIMAP is to dispose your mailbox(es) as a local Maildir(s)."
```

Comment:
We're not adding descriptions to existing formulae for now. It's going to be done in bulk in the near future.